fixes.inc

```diff
@@ -9793,34 +9919,93 @@ native BAD_SetPlayerName(playerid, const name[]) = SetPlayerName;
 			}
 		#endif
 
+		// ==================================
+		//  BEGIN: GetVehicleComponentInSlot
+		// ==================================
+		#if FIX_GetVehicleComponentInSlot
+			switch (FIXES_GetVehicleComponentType(componentid))
+			{
+				#if FIXES_Single
+					case CARMODTYPE_STEREO:
+					{
+						FIXES_gsStereo[vehicleid] = componentid;
+					}
+					case CARMODTYPE_FRONT_BUMPER:
+					{
+						FIXES_gsFrontBumper[vehicleid] = componentid;
+					}
+					case CARMODTYPE_REAR_BUMPER:
+					{
+						FIXES_gsRearBumper[vehicleid] = componentid;
+					}
+					case CARMODTYPE_FRONT_BULLBAR:
+					{
+						FIXES_gsFrontBullbar[vehicleid] = componentid;
+					}
+					case CARMODTYPE_REAR_BULLBAR:
+					{
+						FIXES_gsRearBullbar[vehicleid] = componentid;
+					}
+				#else 
+					case CARMODTYPE_STEREO:
+					{
+						setproperty(5, FIXES_gscStereo, componentid);
+					}
+					case CARMODTYPE_FRONT_BUMPER:
+					{
+						setproperty(5, FIXES_gscFrontBumper, componentid);
```

Comment:
Well we now have the biggest problem.  These don't work for vehicles - you need an array.

My suggestion is to use a separate property ID for each component type, then you can store by vehicle index, encoding the componentid in a string.

Reply:
That means we won't stick to id 5.

Reply:
Like this

https://github.com/pawn-lang/sa-mp-fixes/pull/149/commits/552e09cd2dde6db2be351b8b53402a67f991d9b6
But it seems I can retrieve value sucessfully returning 0 always.